SchedulerJobs/SchedulerJobs.Services/VideoApiService.cs

```diff
@@ -37,14 +39,17 @@ public async Task CloseConference(Guid conferenceId)
         {
             _log.LogTrace($"Close conference by Id {conferenceId}");
             var uriString = _apiUriFactory.ConferenceEndpoints.CloseConference(conferenceId);
-            var response = await _httpClient.GetAsync(uriString).ConfigureAwait(false);
+            var response = await _httpClient.PutAsync(uriString, null).ConfigureAwait(false);
             response.EnsureSuccessStatusCode();
         }
 
         public async Task<List<ConferenceSummaryResponse>> GetOpenConferencesByScheduledDate(DateTime scheduledDate)
         {
             _log.LogTrace($"Getting conference by scheduledDate {scheduledDate}");
-            var uriString = _apiUriFactory.ConferenceEndpoints.GetOpenConferencesByScheduledDate(scheduledDate.ToString());
+
```

Comment:
Why do we provide a date? This implementation will not guarantee anything missed beyond the given day.

Reply:
I think passing in the scheduledDate here is incorrect. The logic should be Get all conferences that have a scheduledDate that is <= to the time function runs. So we should pass in the date-time function runs -14 hours. The query should be scheduledDate <= FromDatetime . This is what the sub task say under VIH-5404.  Valentina, can you please amend the logic and the function names as above. In video API the below need to change as above 


        public async Task<List<Conference>> Handle(GetOpenConferencesByDateTimeQuery query)
        {
            return await _context.Conferences
                .Include("Participants.ParticipantStatuses")
                .Include("ConferenceStatuses")
                .Include("Tasks").AsNoTracking()
                .Where(x => x.ScheduledDateTime <= query.ScheduledDateTime)
                .Where(x => x.State < ConferenceState.Closed)
                .OrderBy(x => x.ScheduledDateTime)
                .ToListAsync();
        }

Reply:
I am agreed with your comments, but I think it was requirement for API (Atiff done this task). Will ask Ru.

Reply:
Agreed, it was Atiff that did the API. But it does not match the Story. Would you mind fixing Valentina? Thanks in advance.

Reply:
The date parameter is removed following the changes in video api